src/pages/main/PersonalContent.jsx

```diff
@@ -7,55 +7,135 @@ const PersonalContent = () => {
     return (
         <div className="p-1 md:mt-4">
             <BasicTitle text="Just for you" />
-            <div className="flex flex-wrap">
             <div className="container mx-auto">
             <div className="flex items-center justify-center w-full h-full py-24 sm:py-8 px-4">
                 {/* Carousel for desktop and large size devices */}
-                <BasicCarousel>
-                <PaddingCardWrapper index_value={0} value="p-3">
```

Comment:
You should move the hardcoded data to a json file (create an array) and import the array into this component, then loop through the array to dynamically create the carousel cards (use arr.map()). It would separate the data and the component as well as create a cleaner codebase